Client/Frontend/Browser/SimpleToast.swift

```diff
@@ -0,0 +1,54 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import Shared
+
+
+struct SimpleToastUX {
+    static let ToastHeight = CGFloat(50)
+    static let ToastAnimationDuration = 0.5
+    static let ToastDefaultColor = UIColor(red: 76.0 / 255, green: 158.0 / 255, blue: 255.0 / 255, alpha: 1)
+    static let ToastFont = UIFont(name: "HelveticaNeue-Thin", size: 14)
+    static let ToastDismissAfter = 0.5
+}
+
+struct SimpleToast {
+
+     func showAlertWithText(text: String) {
+        guard let window = UIApplication.sharedApplication().windows.first,
+        let keyboardHeight = KeyboardHelper.defaultHelper.currentState?.intersectionHeightForView(window) else {
+            return
+        }
+        let startHeight = window.frame.height - keyboardHeight
+        let toast = self.createView(window.frame.width, height: CGFloat(startHeight))
+        toast.text = text
+        window.addSubview(toast)
+        animate(toast)
+    }
+
+    private func createView(width: CGFloat, height: CGFloat) -> UILabel {
+        let toast = UILabel(frame: CGRect(x: 0, y: height, width: width, height: SimpleToastUX.ToastHeight))
+        toast.backgroundColor = SimpleToastUX.ToastDefaultColor
+        toast.textColor = UIColor.whiteColor()
+        toast.font = SimpleToastUX.ToastFont
+        toast.textAlignment = .Center
+        return toast
+    }
+
+    private func animate(toast: UIView) {
+        UIView.animateWithDuration(SimpleToastUX.ToastAnimationDuration,
+            animations: {
+                var frame = toast.frame
+                frame.origin.y = frame.origin.y - SimpleToastUX.ToastHeight
+                toast.frame = frame
+            },
+            completion: { finished in
+                let dispatchTime = dispatch_time(DISPATCH_TIME_NOW, Int64(SimpleToastUX.ToastDismissAfter * Double(NSEC_PER_SEC)))
+                dispatch_after(dispatchTime, dispatch_get_main_queue(), {
+                    toast.removeFromSuperview()
```

Comment:
Can we add a "slide back" animation here so that the view doesn't just abruptly disappear?